config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get 'users/index'
+  get '/users', to: "users#index"
+  get '/users/:id/', to: "users#user"
+  get '/users/:id/posts', to: "users#posts"
+  get '/users/:id/posts/:id', to: "users#post"
+  root 'users#index'
```

Comment:
- [ ] let's not expose unnecessary endpoints. you can just write your routes as follow:
````
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
@uwadonat Thanks